broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -95,52 +141,65 @@ public static BrokerController createBrokerController(String[] args) {
             final BrokerConfig brokerConfig = new BrokerConfig();
             final NettyServerConfig nettyServerConfig = new NettyServerConfig();
             final NettyClientConfig nettyClientConfig = new NettyClientConfig();
-            nettyServerConfig.setListenPort(10911);
+            nettyServerConfig.setListenPort(10911); // Set default broker listen port
             final MessageStoreConfig messageStoreConfig = new MessageStoreConfig();
 
             if (BrokerRole.SLAVE == messageStoreConfig.getBrokerRole()) {
+
+                // Use a conservative value for slave brokers so that consumer groups won't frequently change targeting
+                // brokers when they are consuming messages at the boundary between being probably resident in physical
+                // memory and swapped out.
                 int ratio = messageStoreConfig.getAccessMessageInMemoryMaxRatio() - 10;
                 messageStoreConfig.setAccessMessageInMemoryMaxRatio(ratio);
             }
 
-            if (commandLine.hasOption('p')) {
+            // FIXME: log should not be null
```

Comment:
Creating a JIRA issue for this and having its number as a comment is a good practice.